### PR TITLE
Build ui for scraping component on onboarding

### LINKED ISSUE #10969: Add the option for scraping in the data loaders selection
as a user,
in the onboarding wizard,
in the selection step,
i should be able to select scraping

### PR DESCRIPTION
**Pull request by Palakp41**
Co-authored-by: Iris Scholten <ischolten.is@gmail.com>

Closes #10969 #10783 

_Briefly describe your proposed changes:_
Created new component for scraping which includes the ui to input interval, select bucket dropdown and add urls.
Created actions and reducers for the input, bucket and urls.

kapture 2019-01-14 at 12 26 20

### CODE REVIEW

File: ui/src/onboarding/components/configureStep/Scraper.test.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// Libraries
+import React from 'react'
+import {shallow} from 'enzyme'
+
+// Components
+import {Scraper} from 'src/onboarding/components/configureStep/Scraper'
+
+const setup = (override = {}) => {
```
Comment (ebb-tide) [inline ui/src/onboarding/components/configureStep/Scraper.test.tsx]: thanks for test!

File: ui/src/onboarding/components/configureStep/Scraper.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  MultipleInput,
+  Dropdown,
+  InputType,
+} from 'src/clockface'
+
+interface Props {
+  bucket: string
+  dropdownBuckets: JSX.Element[]
+  onIntervalHandle: (value: string) => void
+  onDropdownHandle: (value: string) => void
```
Comment (ebb-tide) [inline ui/src/onboarding/components/configureStep/Scraper.tsx]: onXHandle sounds like a strange combination of naming conventions to me, would onChooseInterval or onSelectInterval be better?

File: ui/src/onboarding/reducers/dataLoaders.ts [hunk 3]
```diff
@@ -34,6 +34,7 @@ export const INITIAL_STATE: DataLoadersState = {
   precision: WritePrecision.Ns,
   telegrafConfigID: null,
   pluginBundles: [],
+  scraper: {interval: '', bucket: '', urls: []},
```
Comment (ebb-tide) [inline ui/src/onboarding/reducers/dataLoaders.ts]: I wonder if DataLoadersState should move to a more nested state in the future ?
```
{
streaming:...,
line protocol:..., 
scraping:..,
}
```

Comment (ischolten) [reply ui/src/onboarding/reducers/dataLoaders.ts]: i definitely agree with this. after alpha, i believe the wizards will be separating out a lot and that feels like a nice time to make this change

File: ui/src/onboarding/components/configureStep/Scraper.tsx (hunk 2 above)
Comment (Palakp41) [reply ui/src/onboarding/components/configureStep/Scraper.tsx]: Changed it!

**Pull request merged**